Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'react_on_rails', path: '../react_on_rails'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.4'
+gem 'rails', '4.2.5'
```

Comment:
@robwise Let's get rid of this diff noise the next time we regenerate. We'll update the the branch https://github.com/shakacode/react_on_rails-generator-results/tree/fresh-rails-install first.